test/deprecated/io/warnings.good

```diff
@@ -49,3 +65,10 @@ len2b_data
 int(32)
 LOCALE0
 LOCALE0
+0
+false
+0
+false
+false
+false
+internal error: No Regexp Support
```

Comment:
Is your `CHPL_REGEXP` set to `none`?  (You can check this by running `printchplenv`).  I get this output when running the test myself:
```
[Executing diff warnings.good warnings.exec.out.tmp]
74c74,76
< internal error: No Regexp Support
---
> uncaught BadFormatError: bad format: no match (in channel.readf(fmt:string) with path "/dev/null" offset 0)
>   warnings.chpl:46: thrown here
>   warnings.chpl:46: uncaught here
[Error matching program output for deprecated/io/warnings]
```
You'll probably want to either skip this test when CHPL_REGEXP is set to none, or move the relevant calls to a separate test and skip that test under those conditions

Reply:
I suppose that is set to none. This is my environment.
```
CHPL_TARGET_PLATFORM: linux64
CHPL_TARGET_COMPILER: gnu
CHPL_TARGET_ARCH: x86_64
CHPL_TARGET_CPU: native
CHPL_LOCALE_MODEL: flat
CHPL_COMM: none *
CHPL_TASKS: fifo *
CHPL_LAUNCHER: none
CHPL_TIMERS: generic
CHPL_UNWIND: none
CHPL_MEM: cstdlib *
CHPL_ATOMICS: cstdlib
CHPL_GMP: none *
CHPL_HWLOC: none
CHPL_REGEXP: none *
CHPL_LLVM: none *
CHPL_AUX_FILESYS: none
```

Reply:
Ah, looks like you're using the quickstart settings still.  You'll want to be using the normal settings generally when testing changes (depending on what you're working on quickstart can be sufficient, but it really is intended for when someone is just testing out the language rather than using it regularly)